websites/D/DramaCool/presence.ts

```diff
@@ -98,11 +100,7 @@ presence.on("UpdateData", async () => {
 			presenceData.buttons = [
 				{
 					label: strings.viewEpisode,
-					url: document.URL,
-				},
-				{
-					label: strings.viewSeriesButton,
-					url: document.querySelector<HTMLAnchorElement>("div.category a").href,
```

Comment:
Why is this removed?

Reply:
It doesn’t work anymore

Reply:
![IMG_20220803_141826.png](https://user-images.githubusercontent.com/54318514/182549346-afc681d8-0487-440b-b2a8-90f177799584.png)
It's this very element